core/db/migrate/20180202222641_create_store_shipping_methods.rb

```diff
@@ -0,0 +1,10 @@
+class CreateStoreShippingMethods < ActiveRecord::Migration[5.1]
+  def change
+    create_table :spree_store_shipping_methods do |t|
+      t.references :store, foreign_key: { to_table: "spree_stores" }
```

Comment:
I don't think this syntax is valid. It fails under MySQL with

```
Mysql2::Error: Can't create table `solidus_core_test`.`spree_store_shipping_methods` (errno: 150 "Foreign key constraint is incorrectly formed")
```

Reply:
Reading more, `to_table` does seem like a valid argument (I had thought it was not), but for some reason this isn't working.

Reply:
Fails on
```
   (60.4ms)  CREATE TABLE `spree_store_shipping_methods` (`id` bigint NOT NULL AUTO_INCREMENT PRIMARY KEY, `store_id` bigint, `shipping_method_id` bigint, `created_at` datetime NOT NULL, `updated_at` datetime NOT NULL,  INDEX `index_spree_store_shipping_methods_on_store_id`  (`store_id`),  INDEX `index_spree_store_shipping_methods_on_shipping_method_id`  (`shipping_method_id`), CONSTRAINT `fk_rails_1ec4e4430e`
FOREIGN KEY (`store_id`)
  REFERENCES `spree_stores` (`id`)
, CONSTRAINT `fk_rails_24cc4a3cd1`
FOREIGN KEY (`shipping_method_id`)
  REFERENCES `spree_shipping_methods` (`id`)
) ENGINE=InnoDB
```

Reply:
The issue is that this is creating an FK from a bigint to a normal size int.

Removed the FK entirely in in #2596